### DISCUSSION TITLE
How to set up global auth middleware

### DISCUSSION THREAD

**Discussion by toniengelhardt** [Teranode/nuxt-module-alternatives#92] [Q&A] [answered]
In the original `@nuxtjs/auth` module global auth middleware was setup like this in `nuxt.config.ts`:

```ts
router: {
    base: '/',
    middleware: [
      'auth',
    ],
}
```

But redirects don't seem to work with this setup in the `@nuxtjs-alt/auth` module. How to set it up?

**Comment by Denoder** [comment, marked as answer]
```ts
auth: {
    globalMiddleware: true
}
```

**Reply by toniengelhardt** [reply]
Amazing! Thanks so much for this module 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽 🙏🏽
